From c951dd6c826de2dbcb1d239743448bb56753ef96 Mon Sep 17 00:00:00 2001
From: root <vadimp@mellanox.com>
Date: Mon, 27 Nov 2017 15:53:20 +0200
Subject: [v4.9 backport 10/10] config: mellanox configuration

It adds the next configuration flags, used by Mellanox systems:

CONFIG_MLXSW_CORE=m
CONFIG_MLXSW_CORE_HWMON=y
CONFIG_MLXSW_CORE_THERMAL=y
CONFIG_MLXSW_CORE_QSFP=y
CONFIG_MLXSW_PCI=m
CONFIG_MLXSW_I2C=m
CONFIG_MLXSW_MINIMAL=m
CONFIG_I2C_MUX_REG=y
CONFIG_I2C_MUX_MLXCPLD=m
CONFIG_I2C_MLXCPLD=m
CONFIG_SENSORS_IIO_HWMON=y
CONFIG_PMBUS=y
CONFIG_SENSORS_PMBUS=y
CONFIG_SENSORS_LM25066=m
CONFIG_SENSORS_TPS53679=m
CONFIG_SENSORS_UCD9000=m
CONFIG_SENSORS_UCD9200=m
CONFIG_LEDS_MLXREG=m
CONFIG_MLX_PLATFORM=m
CONFIG_MELLANOX_PLATFORM=y
CONFIG_MLXREG_HOTPLUG=m
CONFIG_MLXREG_IO=m
CONFIG_MAX1363=y

Signed-off-by: Vadim Pastrenak <vadimp@melanox.com>
---
 debian/build/build_amd64_none_amd64/.config | 28 +++++++++++++++++++++++-----
 1 file changed, 23 insertions(+), 5 deletions(-)

diff --git a/debian/build/build_amd64_none_amd64/.config b/debian/build/build_amd64_none_amd64/.config
index 88a71ef..f1930a1 100644
--- a/debian/build/build_amd64_none_amd64/.config
+++ b/debian/build/build_amd64_none_amd64/.config
@@ -2632,7 +2632,13 @@ CONFIG_MLX4_DEBUG=y
 CONFIG_MLX5_CORE=m
 CONFIG_MLX5_CORE_EN=y
 CONFIG_MLX5_CORE_EN_DCB=y
-# CONFIG_MLXSW_CORE is not set
+CONFIG_MLXSW_CORE=m
+CONFIG_MLXSW_CORE_HWMON=y
+CONFIG_MLXSW_CORE_THERMAL=y
+CONFIG_MLXSW_CORE_QSFP=y
+CONFIG_MLXSW_PCI=m
+CONFIG_MLXSW_I2C=m
+CONFIG_MLXSW_MINIMAL=m
 CONFIG_NET_VENDOR_MICREL=y
 # CONFIG_KS8842 is not set
 # CONFIG_KS8851 is not set
@@ -3565,7 +3571,8 @@ CONFIG_I2C_MUX=m
 # CONFIG_I2C_MUX_PCA9541 is not set
 # CONFIG_I2C_MUX_PCA954x is not set
 # CONFIG_I2C_MUX_PINCTRL is not set
-# CONFIG_I2C_MUX_REG is not set
+CONFIG_I2C_MUX_REG=y
+CONFIG_I2C_MUX_MLXCPLD=m
 CONFIG_I2C_HELPER_AUTO=y
 CONFIG_I2C_SMBUS=m
 CONFIG_I2C_ALGOBIT=m
@@ -3632,6 +3639,7 @@ CONFIG_I2C_VIPERBOARD=m
 #
 # Other I2C/SMBus bus drivers
 #
+CONFIG_I2C_MLXCPLD=m
 CONFIG_I2C_STUB=m
 # CONFIG_I2C_SLAVE is not set
 # CONFIG_I2C_DEBUG_CORE is not set
@@ -3879,7 +3887,7 @@ CONFIG_SENSORS_G760A=m
 # CONFIG_SENSORS_HIH6130 is not set
 CONFIG_SENSORS_IBMAEM=m
 CONFIG_SENSORS_IBMPEX=m
-# CONFIG_SENSORS_IIO_HWMON is not set
+CONFIG_SENSORS_IIO_HWMON=y
 CONFIG_SENSORS_I5500=m
 CONFIG_SENSORS_CORETEMP=m
 CONFIG_SENSORS_IT87=m
@@ -3932,7 +3940,12 @@ CONFIG_SENSORS_NCT6775=m
 # CONFIG_SENSORS_NCT7802 is not set
 # CONFIG_SENSORS_NCT7904 is not set
 CONFIG_SENSORS_PCF8591=m
-# CONFIG_PMBUS is not set
+CONFIG_PMBUS=y
+CONFIG_SENSORS_PMBUS=y
+CONFIG_SENSORS_LM25066=m
+CONFIG_SENSORS_TPS53679=m
+CONFIG_SENSORS_UCD9000=m
+CONFIG_SENSORS_UCD9200=m
 # CONFIG_SENSORS_SHT15 is not set
 CONFIG_SENSORS_SHT21=m
 # CONFIG_SENSORS_SHT3x is not set
@@ -5904,6 +5917,7 @@ CONFIG_LEDS_MENF21BMC=m
 #
 # CONFIG_LEDS_BLINKM is not set
 # CONFIG_LEDS_MLXCPLD is not set
+CONFIG_LEDS_MLXREG=m
 
 #
 # LED Triggers
@@ -6483,10 +6497,14 @@ CONFIG_PVPANIC=m
 CONFIG_INTEL_PMC_IPC=m
 CONFIG_SURFACE_PRO3_BUTTON=m
 # CONFIG_INTEL_PUNIT_IPC is not set
+CONFIG_MLX_PLATFORM=m
 CONFIG_CHROME_PLATFORMS=y
 CONFIG_CHROMEOS_LAPTOP=m
 CONFIG_CHROMEOS_PSTORE=m
 CONFIG_CROS_KBD_LED_BACKLIGHT=m
+CONFIG_MELLANOX_PLATFORM=y
+CONFIG_MLXREG_HOTPLUG=m
+CONFIG_MLXREG_IO=m
 CONFIG_CLKDEV_LOOKUP=y
 CONFIG_HAVE_CLK_PREPARE=y
 CONFIG_COMMON_CLK=y
@@ -6621,7 +6639,7 @@ CONFIG_MMA9553=m
 # CONFIG_INA2XX_ADC is not set
 # CONFIG_LTC2485 is not set
 # CONFIG_MAX1027 is not set
-# CONFIG_MAX1363 is not set
+CONFIG_MAX1363=y
 # CONFIG_MCP320X is not set
 # CONFIG_MCP3422 is not set
 # CONFIG_NAU7802 is not set
-- 
2.11.0

